layouts/default.vue

```diff
@@ -1,18 +1,86 @@
 <template>
   <div>
-    <HeaderBanner />
-    <div class="lead">
-      <span class="logo">Arctic-EDS</span><br>
-      <span class="name">Arctic Environmental and Engineering Data and Design Support System</span>
+    <noscript>
```

Comment:
If this site used multiple `layouts` we'd need to repeat this (you can put an `app.html` in the root directory of the app instead) but that's no big deal.  Just be aware when doing this in a Nuxt app that you kinda might need to go beyond `layout` and figure out the right place to inject it (different in Nuxt 2 / 3).